internal/querycoord/task_scheduler.go

```diff
@@ -662,7 +662,7 @@ func (scheduler *TaskScheduler) scheduleLoop() {
 			childTasks := triggerTask.getChildTask()
 			if len(childTasks) != 0 {
 				// process loadSegment before watchDmChannel, avoid delete not taking effect
-				highPriorityTasks, lowPriorityTasks := sortInternalTaskByPriority(childTasks, commonpb.MsgType_LoadSegments)
+				highPriorityTasks, lowPriorityTasks := sortInternalTaskByPriority(childTasks, commonpb.MsgType_LoadSegments, commonpb.MsgType_WatchDmChannels)
```

Comment:
There should be no dependency between watchQuerychannel and watchDelta/dm channel